lib/map.jsx

```diff
@@ -0,0 +1,28 @@
+var React = require('react');
+
+var MapComponent = React.createClass({
+
+    propTypes: {
+      accessToken: React.PropTypes.string.isRequired,
+      mapId: React.PropTypes.string.isRequired,
+    },
+
+    componentDidMount: function () {
+      console.log('here1')
+      // Provide your access token
+      L.mapbox.accessToken = this.props.accessToken;
```

Comment:
Rather than setting the global `accessToken`, I think it's better to pass it into the [`L.mapbox.map`](https://www.mapbox.com/mapbox.js/api/v2.1.5/l-mapbox-map/) constructor, so as to avoid messing with global state.